sparta/src/Simulation.cpp

```diff
@@ -1110,6 +1110,7 @@ void Simulation::saveReports()
             formatters_in_use = report_repository_->getFormattersByFilename();
             db::SimInfoSerializer serializer(SimulationInfo::getInstance());
             serializer.serialize(*stats_db_);
+            std::cout << "SimDB file is: " << stats_db_->getDatabaseFile() << std::endl;
```

Comment:
Does this print the location as well, or just the name?

Reply:
The full path and file name.
```
Saving reports...
Automatic Simulation Summary [0,200001]  
  scheduler
          ticks                                                    = 200000 
          picoseconds                                       = 200000 
...
  [simdb] Validation of database-generated auto summary is currently unavailable
SimDB file is: /tmp/b487ac21-e0e9-4035-afcb-7a8618566367.db 
```
That line about "Validation of database..." is extraneous and probably should be deleted.

And, as I look at this more closely, I see I caused a side effect (print to stdout) in a lambda function definition.  I can move that statement to a later point where the lambda is used.  Can I withdraw my pull request?

Reply:
You could git rebase -i to clean up commits, or you could switch the "base branch" of the PR, supposedly using the edit button next to the PR title.

Reply:
I can also close the pull request without merge.  Let me know what you want to do.

Reply:
Never mind.  My brain took the afternoon off.  What I thought was happening isn't how lambdas work.  I think this code is OK.